src/index.js

```diff
@@ -443,8 +443,8 @@ class MyGame extends Phaser.Scene
 
 const config = {
     type: Phaser.CANVAS,
-    width: window.innerWidth - 20,
-    height: window.innerHeight - 70,
+    width: 1400,
+    height: 760,
```

Comment:
it's not desired to hardcode window size, it has to work on different window sizes + there is some margin left for now while some UI is in HTML (the plan is to move everything in game itself to Phaser canvas).